easybuild/easyblocks/s/serf.py

```diff
@@ -0,0 +1,69 @@
+##
+# Copyright 2009-2015 Ghent University
+#
+# This file is part of EasyBuild,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://vscentrum.be/nl/en),
+# the Hercules foundation (http://www.herculesstichting.be/in_English)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# http://github.com/hpcugent/easybuild
+#
+# EasyBuild is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation v2.
+#
+# EasyBuild is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with EasyBuild.  If not, see <http://www.gnu.org/licenses/>.
+##
+"""
+EasyBuild support for building and installing Serf, implemented as an easyblock
+
+@author: Balazs Hajgato (Free University Brussels (VUB))
+"""
+
+import os
+from easybuild.framework.easyblock import EasyBlock
+from easybuild.tools.run import run_cmd
+
+class EB_Serf(EasyBlock):
+    """Support for building/installing Serf."""
+
+    def configure_step(self):
+        """
+        No configure step for Serf
+        """
+        pass
+
+    def build_step(self):
+        """
+        Build Serf using 'scons APR=/path/to/apr APU=/path/to/apu OPENSSL=/openssl/base PREFIX=/path/to/prefix
+        """
+        cmd = "scons %s PREFIX=%s" % (self.cfg['buildopts'], self.installdir)
+        (out, _) = run_cmd(cmd, log_all=True, simple=False)
+
+        return out
```

Comment:
typically there's little reason to return the output here, it's not used 'upstream'

so, replace by

```python
run_cmd(cmd, log_all=True)
```